.github/workflows/changesets.yml

```diff
@@ -150,7 +150,7 @@ jobs:
           content="${content//$'\n'/'%0A'}"
           content="${content//$'\r'/'%0D'}"
           # end of handling multi line json
-          echo "::set-output name=packageJson::$content"
+          echo "packageJson=$content" >> $GITHUB_OUTPUT
```

Comment:
Following this deprecation instruction https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/